test/EFCore.PG.FunctionalTests/Query/ArrayArrayQueryTest.cs

```diff
@@ -363,13 +361,74 @@ public void Index_text_with_constant()
 
         #endregion
 
+        #region ArrayToString
+
+        [Fact]
+        public void String_Join_Array()
+        {
+            using var ctx = CreateContext();
+
+            var found = ctx.SomeEntities.FirstOrDefault(x => string.Join(" ", x.SomeArray) == "3 4");
+            Assert.NotNull(found);
+            Assert.Contains("array_to_string", Fixture.TestSqlLoggerFactory.Sql);
+        }
+
+
+        [Fact]
+        public void Array_to_tsvector_concat()
```

Comment:
I don't see what this test actually tests... the translation of string.Join is already tested above (in String_Join_Array), wrapping that in ToTsVector doesn't actually test anything here...